src/organisms/Accordion/index.js

```diff
@@ -67,12 +70,17 @@ Accordion.propTypes = {
       title: PropTypes.string.isRequired,
       body: PropTypes.string.isRequired
     })
-  )
+  ),
+  /**
+   * Whether or not to number the accordion items. Defaults to `true`.
+   */
+  numberedList: PropTypes.bool
 };
 
 Accordion.defaultProps = {
   openText: 'View',
-  closeText: 'Close'
+  closeText: 'Close',
+  numberedList: true
```

Comment:
Since this defaults to true we can assume that the user wouldn't need to specify it in props. Would it make more sense to rename this field `disableNumberedList` 
i.e `<Accordion disableNumberedList />`

Reply:
Yeah I was kinda on the fence about which way around it should. I think that makes sense tho, I'll switch it around 👍